Tests/Set-BBServerCommitBuildStatus.Tests.ps1

```diff
@@ -14,7 +14,7 @@
 Set-StrictMode -Version 'Latest'
 
 & (Join-Path -Path $PSScriptRoot -ChildPath 'Initialize-BitbucketServerAutomationTest.ps1' -Resolve)
-& (Join-Path -Path $PSScriptRoot -ChildPath '..\LibGit2\Import-LibGit2.ps1' -Resolve)
+& (Join-Path -Path $PSScriptRoot -ChildPath '..\PSModules\GitAutomation\*\Import-GitAutomation.ps1' -Resolve)

```

Comment:
Change to `Import-Module -Path (Join-Path -Path $PSScriptRoot -ChildPath '..\PSModules\GitAutomation') -Force`.